sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/ScalaJSBundlerPlugin.scala

```diff
@@ -553,6 +535,8 @@ object ScalaJSBundlerPlugin extends AutoPlugin {
         ReloadWorkflowTasks.webpackTask(fastOptJS)
       }.value,
 
+      installWebpackDevServer := npmUpdate.value,
```

Comment:
Do we need to keep this task at all? It is used only to scope the version of `webpack-dev-server` to use, I guess.